### PR TITLE
fix: support evaluating TypeScript files by default. fixes #27

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: src/babel/module.js [hunk 1]
```diff
@@ -58,10 +59,35 @@ class Module {
     this.require.resolve = this.resolve.bind(this);
     this.require.ensure = NOOP;
     this.require.cache = cache;
+
+    // We support following extensions by default
+    this.extensions = ['.json', '.js', '.ts', '.tsx'];
```
Comment (thymikee) [inline src/babel/module.js]: add `.jsx` as well? I'm cool with not adding that and forcing others to just use `.js` though :D

File: src/babel/module.js [hunk 2]
```diff
@@ -86,15 +112,16 @@ class Module {
       // we would end up in infinite loop with cyclic dependencies
       cache[filename] = m;
 
-      if (/\.(js|json)$/.test(filename)) {
-        // For JS/JSON files, we need to read the file first
+      if (/\.(json|js|tsx?)$/.test(filename)) {
```
Comment (thymikee) [inline src/babel/module.js]: I think it's safer to generate this regex based on `this.extensions`, but that's not gonna change too often, so feel free to ignore this advice

File: src/babel/module.js (hunk 1 above)
Comment (satya164) [reply src/babel/module.js]: death to `.jsx`

Comment (satya164) [reply src/babel/module.js]: @thymikee I would if I could. But due to syntax ambiguity with deprecated typecast syntax, TypeScript relies on the `.tsx` extension to understand JSX!

**Pull request merged**